test/parallel/test-http-set-timeout-server.js

```diff
@@ -21,6 +21,12 @@
 
 'use strict';
 const common = require('../common');
+
+if (!common.hasCrypto) {
```

Comment:
Yes, this is unnecessary in the `http` test but required in the `https` test. It should be removed from this test.